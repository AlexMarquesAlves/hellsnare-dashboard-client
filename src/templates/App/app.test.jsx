import { screen } from '@testing-library/react';
import { App } from '.';
import { renderTheme } from '../../styles/render-theme';

it('should render heading', () => {
  renderTheme(<App>Test</App>);
  expect(screen.getByRole('heading', { name: 'Test' })).toMatchSnapshot();
});
