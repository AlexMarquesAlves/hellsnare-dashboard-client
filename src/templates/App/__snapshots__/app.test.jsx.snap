// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render heading 1`] = `
.c0 {
  background: #191F45;
}

<h1
  class="c0"
>
  Test
</h1>
`;
